test/libyul/Parser.cpp

```diff
@@ -573,9 +573,7 @@ BOOST_AUTO_TEST_CASE(customSourceLocations_reference_original_sloc)
 	BOOST_REQUIRE(holds_alternative<VariableDeclaration>(result->statements.at(0)));
 	VariableDeclaration const& varDecl = get<VariableDeclaration>(result->statements.at(0));
 
-	// -1 points to original source code, which in this case is `sourceText` (which is also
-	// available via `0`, that's why the first @src is set to `1` instead.)
-	CHECK_LOCATION(varDecl.debugData->location, sourceText, 10, 20);
+	CHECK_LOCATION(varDecl.debugData->location, "", 10, 20);
```

Comment:
This is now an empty string as source because https://github.com/ethereum/solidity/pull/11646/files#diff-8928f8175b3fd5c14f591cde7c0a56d709fba1dc45e6bdc5b825c6be124aad7dR135 is hit and `CHECK_LOCATION` evaluates source text to empty string if location is null.

LGTM